doc/chromebooks/setting_gbb_flags.md

```diff
@@ -0,0 +1,200 @@
+# Why?
+
+![screw](./assets/wp-screw/lenovo10e.jpg)
+
+setting gbb flags will keep your device in developer mode with option to boot from usb at any time even if your battery ryns dry
+
+these flags are usually protected by hardware write protection so you will need to disable it first
+
+**But before**
+
+in order to set gbb flags you need to [enable developer mode](./preparing-chromebook.md) on your device
+
+# Disabling hardware write protection
+
+hardware write protection can be dissabled in a few ways
+
+## Removing a screw or shorting a circut (older chromebooks)
```

Comment:
this maybe makes more clear that this is the way to go for older chromebooks
```suggestion
## Removing a screw or shorting a circut (required for older chromebooks)
```

Reply:
maybe "only older chromebooks" instead, the guide alrea says that disabled wp is required to proceed and never chromebooks don't have this

Reply:
lets keep it as you had it then - fine with me as well